repl/repl.go

```diff
@@ -223,7 +232,9 @@ func Interactive(options Options) int {
 	term.NewHistory(options.MaxHistory)
 	_ = term.SetHistoryFile(options.HistoryFile)
 	_ = AutoLoad(s, options) // errors already logged
-	// Regular expression for "!nn" to run history command nn.
+	if options.PreInput != nil {
```

Comment:
No test for this?

Reply:
the test (use) is in the bot but I can probably should add one here

Reply:
added, ptal